packages/test-helpers/src/router.rs

```diff
@@ -0,0 +1,112 @@
+use apollo_cw_asset::AssetInfoUnchecked;
+use cosmwasm_std::Addr;
+use cw_dex_router::helpers::{CwDexRouter, CwDexRouterUnchecked};
+use cw_dex_router::msg::InstantiateMsg;
+use cw_dex_router::operations::SwapOperationsListUnchecked;
+use cw_it::cw_multi_test::ContractWrapper;
+use cw_it::robot::TestRobot;
+use cw_it::test_tube::{Account, Module, SigningAccount, Wasm};
+use cw_it::traits::CwItRunner;
+use cw_it::{ContractType, TestRunner};
+
+#[cfg(feature = "osmosis-test-tube")]
+use cw_it::Artifact;
+
+pub const CW_DEX_ROUTER_WASM_NAME: &str = "cw_dex_router_osmosis.wasm";
+
+pub struct CwDexRouterRobot<'a> {
```

Comment:
this is used in many of our vaults. this is ok for now but long term we should put it in the dex-router crate asA helper

Reply:
Agree.